File: csharp/SlimDownYourAggregates.Tests/Slimmed/Events/BookMovedToPrintingEvent.cs
using SlimDownYourAggregates.Tests.Slimmed.Core;
using SlimDownYourAggregates.Tests.Slimmed.Entities;

namespace SlimDownYourAggregates.Tests.Slimmed.Events;

public record BookMovedToPrintingEvent
(
    BookId BookId
): IDomainEvent;

File: java/src/main/java/io/eventdriven/slimdownaggregates/slimmed/events/BookMovedToPrintingEvent.java
package io.eventdriven.slimdownaggregates.slimmed.events;

import io.eventdriven.slimdownaggregates.slimmed.core.IDomainEvent;
import io.eventdriven.slimdownaggregates.slimmed.entities.BookId;

public record BookMovedToPrintingEvent(
  BookId bookId
) implements IDomainEvent {
}
